packages/repository-json-schema/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@loopback/repository-json-schema",
+  "version": "4.0.0-alpha.1",
+  "description": "Converts TS classes into JSON Schemas using TypeScript's reflection API",
+  "engines": {
+    "node": ">=6"
+  },
+  "scripts": {
+    "build": "npm run build:dist",
+    "build:current": "lb-tsc",
+    "build:dist": "lb-tsc es2017",
+    "build:dist6": "lb-tsc es2015",
+    "build:apidocs": "lb-apidocs",
+    "clean": "lb-clean loopback-json-schema*.tgz dist dist6 package api-docs",
+    "prepare": "npm run build && npm run build:apidocs",
+    "pretest": "npm run build:current",
+    "test": "lb-dist mocha --opts ../../test/mocha.opts 'dist/test/unit/**/*.js' 'dist/test/integration/**/*.js' 'dist/test/acceptance/**/*.js'",
```

Comment:
`mocha.opts` was moved to a different location (see https://github.com/strongloop/loopback-next/pull/876), please rebase your PR and update this line accordingly.